README.md

```diff
@@ -4,25 +4,27 @@ Django (Python) website for RedCraft
 
 ## Setup the project
 
-First, creates the `.env` file with the help of `.env.example`.
+First, copy `.env.example` to `.env` and change the configuration as necessary in `.env`
 
-Create your folder cache.
+Make sure the directory specified in `FILE_CACHE_PATH` exists
 
-Create a DB `redcraft` in your mysql server.
+Create a DB named `redcraft` on your MySQL server.
 
-Create your virtual env: `python3 -m venv env`
-And start your venv:
-Window: `"env/Scripts/activate"`
-Linux: `source env/bin/activate`
+### Linux
 
+Run `./setup.sh`
 
+### Windows
 
+Create your virtual env: `python3 -m venv env`
+And start your venv: `"env/Scripts/activate"`
 Install  requirements: `pip install -r requirements.txt`
-
-Make migration: `python manage.py makmigrations`
+Make migration: `python manage.py makemigrations`
 Migrate DB: `python manage.py migrate`
-
 Compile sass: `python manage.py sass website/static/website/scss/ website/static/website/css/`
```

Comment:
```suggestion
Compile sass: `python manage.py sass website/static/website/scss/ website/static/website/css/` --watch
```
For automatic scss file compilation

Reply:
Already documented in the section after :)